lib/node_modules/@stdlib/math/base/special/roundsd/README.md

```diff
@@ -85,36 +85,106 @@ logEachMap( 'x: %0.4f. y: %d. Rounded: %0.4f.', x, 5, roundsd );
 
 <!-- /.examples -->
 
-<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
-
-<section class="related">
+<!-- C interface documentation. -->
 
 * * *
 
-## See Also
+<section class="c">
+
+## C APIs
 
--   <span class="package-name">[`@stdlib/math/base/special/ceilsd`][@stdlib/math/base/special/ceilsd]</span><span class="delimiter">: </span><span class="description">round a numeric value to the nearest number toward positive infinity with N significant figures.</span>
--   <span class="package-name">[`@stdlib/math/base/special/floorsd`][@stdlib/math/base/special/floorsd]</span><span class="delimiter">: </span><span class="description">round a numeric value to the nearest number toward negative infinity with N significant figures.</span>
--   <span class="package-name">[`@stdlib/math/base/special/round`][@stdlib/math/base/special/round]</span><span class="delimiter">: </span><span class="description">round a numeric value to the nearest integer.</span>
--   <span class="package-name">[`@stdlib/math/base/special/truncsd`][@stdlib/math/base/special/truncsd]</span><span class="delimiter">: </span><span class="description">round a numeric value to the nearest number toward zero with N significant figures.</span>
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
 
 </section>
 
-<!-- /.related -->
+<!-- /.intro -->
 
-<!-- Section for all links. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+<!-- C usage documentation. -->
 
-<section class="links">
+<section class="usage">
 
-<!-- <related-links> -->
+### Usage
+
+```c
+#include "stdlib/math/base/special/roundsd.h"
+```
+
+#### stdlib_base_roundsd( x, n, b )
+
+Rounds a `numeric` value to the nearest `number` with `n` significant figures.
```

Comment:
```suggestion
Rounds a double-precision floating-point number to the nearest value with `n` significant figures.
```

Reply:
It's likely that we should update the description elsewhere throughout this package.